src/coreComponents/physicsSolvers/SolverBase.cpp

```diff
@@ -113,15 +113,16 @@ bool SolverBase::checkModelNames( array1d< string > & modelNames,
   }
 
   GEOSX_ERROR_IF_NE_MSG( modelNames.size(), m_targetRegionNames.size(),
-                         getName() << ": invalid number of values in " << attribute << " attribute "
-                                                                                       "(expected one model name per target region, or one value for all regions)" );
+                         GEOSX_FMT( "{}: invalid number of values in attribute '{}' "
+                                    "(expected one model name per target region, or one value for all regions)",
+                                    getName(), attribute ) );
```

Comment:
You see here, I think that the `GEOSX_FMT` is more difficult to read than the `std` version.
On the left, there was an issue with the indentation, for sure. But we could have fixed it.

Reply:
I sort of disagree, but without a strong/technical argument. It all comes down to personal preference, I guess. I prefer to scan the entire message first to get an idea of what's being reported, then look at the specific arguments. Or maybe this preference is reinforced by the prevalence of `printf` descendants in programming languages; I don't know at this point.

Reply:
> I prefer to scan the entire message first to get an idea of what's being reported, then look at the specific arguments.

The idea is that you rarely cannot really understand a sentence when words are missing. Language is not a function! 😉 
https://en.wikipedia.org/wiki/The_Smurfs